Source/init.cpp

```diff
@@ -147,11 +147,18 @@ void init_archives()
 	init_get_file_info();
 
 	std::vector<std::string> paths;
-	paths.reserve(2);
+	paths.reserve(3);

```

Comment:
technically yes but no big deal imo

Reply:
Can't we just set it to 5 for every build? Otherwise it will always re-allocate on linux. If you are touching this at all, I feel like we should at least use a more sensible value here.

Reply:
Sure, done